packages/js-client/src/types.ts

```diff
@@ -0,0 +1,50 @@
+import {
+  ContextParams,
+  ContextState,
+  Pagination,
+  OverriddenState,
+  VersionTag,
+} from '@aragon/sdk-client-common';
+
+export type SimpleStorageContextState = ContextState & {
+  // extend the Context state with a new state for storing
+  // the new parameters
+  simpleStoragePluginAddress: string;
+  simpleStorageRepoAddress: string;
+};
+
+export type SimpleStorageOverriddenState = OverriddenState & {
+  simpleStoragePluginAddress: boolean;
+  simpleStorageRepoAddress: boolean;
+};
```

Comment:
Avoid human error by automating the `boolean` type remap

```suggestion
export type SimpleStorageOverriddenState = OverriddenState & {
  [key in keyof SimpleStorageContextState]: boolean;
};
```